hpack/table.py

```diff
@@ -23,6 +23,12 @@ def table_entry_size(name, value):
     return 32 + len(name) + len(value)
 
 
+class _HeaderNameSearchResult(object):
+    def __init__(self):
+        self.min = None
+        self.positions = {}
```

Comment:
Could you explain more, I did not quite understand what was meant. PS. Sorry for my english.